.github/workflows/ci-workflow.yml

```diff
@@ -75,7 +76,7 @@ env:
     -pl -nifi-toolkit/nifi-toolkit-assembly
 
 concurrency:
-  group: ${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Can you provide some background for this particular change?

Reply:
Adding a `concurrency` entry for the new Workflows (and existing `system-tests`) seemed sensible to avoid running duplicate Workflows for PRs that trigger them (the `cron` schedule, of course, should not result in duplicate runs)

Having a `group` of just `${{ github.ref }}` means that the Workflows cancel each other because they're all running in the same `group`. Adding the `${{ github.workflow }}` name prefix means that a Workflow can only cancel an existing run of itself. There's not really a *need* to add the prefix to the `ci-workflow` Workflow definition, as it could be left as the only Workflow without the prefix, meaning it would still be in a `group` of its own. But as I was upting this Workflow definition for other reasons, then making this consistent with all other NiFi Workflows seemed sensible for the sake of future maintenance.